src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AsyncRewriter.AsyncIteratorRewriter.cs

```diff
@@ -31,7 +33,8 @@ private sealed class AsyncIteratorRewriter : AsyncRewriter
             {
                 Debug.Assert(method.IteratorElementType != null);
 
-                // PROTOTYPE(async-streams): Why does AsyncRewriter have logic to ignore accessibility?
```

Comment:
:memo: removed the unnecessary `_ignoreAccessibility` field from `AsyncRewriter`.